src/containers/PackingOverview/RemindersPage/RemindersPage.js

```diff
@@ -17,60 +19,139 @@ const RemindersPage = props => {
     bag_id
   } = props;
 
-  const [todoList, setTodoList] = useState(null);
+  const [todoList, setTodoList] = useState([]);
+  const [todoListId, setTodoListId] = useState(null);
+  const [shoppingListId, setShoppingListId] = useState(null);
   const [shoppingList, setShoppingList] = useState([]);
   const [alertDisplay, setAlertDisplay] = useState(false);
   const [itemInput, setItemInput] = useState("");
+  const [todos, setTodos] = useState([]);
+  const [todoDisplay, setTodoDisplay] = useState(false);
+  const [todoInput, setTodoInput] = useState("");
 
   useEffect(() => {
-    console.log("lists", lists);
+    console.log(lists, "lists");
+    console.log("selected", selectedList);
     if (!lists.length) return;
     for (let list of lists) {
       getList(list);
     }
   }, [lists]);
 
-  const createList = () => {
-    if (lists.list_type === "Todo List" || "Shopping List") {
-      setAlertDisplay(true);
-    } else {
-      axios({
-        method: "post",
-        url: "http://localhost:5000/todolist/",
-        data: {
-          name: "",
-          trip_id,
-          list_type: selectedList
-        }
+  useEffect(() => {
+    console.log(shoppingList, "s list");
+    console.log(shoppingListId, "s id");
+    console.log(todoList, "tdl");
+    console.log(todoListId, "tdl id");
```

Comment:
Are you going to clean these up before merging? 😄

Reply:
I definitely will! I just needed them (well, half of them) still for a feature I'm trying to get working.